src/items/itemsLandingPage.component.tsx

```diff
@@ -68,32 +98,30 @@ function ItemsLandingPage() {
         }}
         item
```

Comment:
```suggestion
        container
        item
```
As it contains items